@shared/api/types.ts

```diff
@@ -196,7 +196,7 @@ export type AssetType = Types.AssetBalance | Types.NativeBalance | TokenBalance;
 export type TokenBalances = SorobanBalance[];
 
 /* eslint-disable camelcase */
-export type HorizonOperation = any;
+export type HorizonOperation = Horizon.ServerApi.OperationRecord;
```

Comment:
the no unsafe arg lint rules forces to to stop passing any/unknown to functions, so this was needed but can be cleaned up further.